data/aerial-photography/index.html

```diff
@@ -27,12 +28,12 @@ <h3 id="GOOGLE">
     </a>
     <ul class="dotless">
       <li>Natural Color (RGB)
-      <li>~6" pixels
+      <li>~6-inch pixels
```

Comment:
K, this comment applies to the whole page really: What do we want to do about spelling out numbers and measurement terms (e.g., in, ft, cm, etc.)? Sometimes we spell out numbers; sometimes we use numerals. Sometimes we use "inch"; sometimes we use in or "...

What do we want to do? (It was the "cm" being spelled out as "centimeters" that kind of threw me off those. Seems strange to spell it out, but then again, consistency is important...

Reply:
let's go with

-inch
-foot
-meter
-cm